daffodil-io/src/main/scala/org/apache/daffodil/io/DataOutputStreamImplMixin.scala

```diff
@@ -149,7 +149,7 @@ trait DataOutputStreamImplMixin extends DataStreamCommonState
    * of the former starting bit pos so as to be able to
    * convert relative positions to absolute positions correctly.
    */
-  protected final def maybeAbsStartingBitPos0b = {
+  final def maybeAbsStartingBitPos0b = {
```

Comment:
Now that this method is public, it should have an explicit return type annotation per Scala's style [guide](https://docs.scala-lang.org/style/types.html).